### ISSUE TITLE
Learning Curves

### ISSUE THREAD

**Issue by lean2206** [matterport/Mask_RCNN#2131]
Could anyone tell me what´s going on during training with the validation loss curve? why it´s so unstable while the training loss curve is slowly converging but quite stable?. Does it has anything to do with pathological curvatures? 

https://blog.paperspace.com/intro-to-optimization-momentum-rmsprop-adam/

![image](https://user-images.githubusercontent.com/50563528/79645053-48c83280-8183-11ea-9030-0caabb586af6.png)

**Comment by Lincoln93** [comment]
have you used default optimizer? SGD?

**Comment by lean2206** [comment]
Yes, i didn´t even modify the configuration class (the iterations only). I think i´ve found the problem in my validation dataset, perhaps there are more examples of one class than the others (i have only three classes).

https://machinelearningmastery.com/learning-curves-for-diagnosing-machine-learning-model-performance/

The learning curves i´ve obtain looks like the last example in that post

**Comment by andreap9** [comment]
> Yes, i didn´t even modify the configuration class (the iterations only). I think i´ve found the problem in my validation dataset, perhaps there are more examples of one class than the others (i have only three classes).
> 
> https://machinelearningmastery.com/learning-curves-for-diagnosing-machine-learning-model-performance/
> 
> The learning curves i´ve obtain looks like the last example in that post

Have you solved the problem? I am having the same issue and I am not sure if the problem is the validation set or if I need more images in the dataset.

**Comment by Kolmobatem** [comment]
> Could anyone tell me what´s going on during training with the validation loss curve? why it´s so unstable while the training loss curve is slowly converging but quite stable?. Does it has anything to do with pathological curvatures?
> 
> https://blog.paperspace.com/intro-to-optimization-momentum-rmsprop-adam/
> 
> ![image](https://user-images.githubusercontent.com/50563528/79645053-48c83280-8183-11ea-9030-0caabb586af6.png)

Hi ! How do you plot these learning curves ?
